app/gui/view/graph-editor/src/component/breadcrumbs.rs

```diff
@@ -53,31 +53,19 @@ const TEXT_Y_OFFSET: f32 = 2.0;
 
 
 
-// ========================
-// === RelativePosition ===
-// ========================
-
-#[derive(Debug, Clone, Copy)]
-enum RelativePosition {
-    Left,
-    Right,
-}
-
-
-
 // ===========
 // === Frp ===
 // ===========
 
 ensogl::define_endpoints! {
     Input {
-        /// Pushes a new breadcrumb after the selected breadcrumb. If the pushed breadcrumb already
-        /// exists as the next one of the stack, it's just selected. If the next breadcrumb isn't
-        /// the breadcrumb being pushed, any existing breadcrumb following the currently selected
-        /// breadcrumb is removed from the panel.
-        push_breadcrumb             (Option<LocalCall>),
-        /// Pops the selected breadcrumb.
-        pop_breadcrumb              (),
+        /// Pushes new breadcrumbs after the selected breadcrumb. Any breadcrumbs that are already
+        /// part of the stack are ignored. If the next breadcrumb isn't part of the breadcrumbs
+        /// being pushed, any existing breadcrumb following the currently selected breadcrumb is
+        /// removed from the panel.
+        push_breadcrumbs            (Vec<LocalCall>),
```

Comment:
Do not use `Vec` in FRP API, this will clone it on every event, which can affect performance. Use `ImVec` (I think we have somewhere `ImVec` = `Rc<Vec<..>>`

Reply:
Hmm, I can't seem to find that anywhere in the code. Where is it?